protos/flyteidl/core/workflow.proto

```diff
@@ -249,4 +253,37 @@ message WorkflowTemplate {
 message TaskNodeOverrides {
     // A customizable interface to convey resources requested for a task container. 
     Resources resources = 1;
+
+    // Boolean that indicates if caching should be enabled
+    oneof cache_value {
```

Comment:
For all of these `oneof`s we can also use `google.protobuf.BoolValue`, `google.protobuf.Int32Value`, etc which differentiate between being set and unset in addition to the values.

Reply:
Whats the concern of oneof?